tools/Uno.Sdk.Updater/Uno.Sdk.Updater.csproj

```diff
@@ -7,6 +7,7 @@
 		<Nullable>enable</Nullable>
 		<TemplatesSourceDirectory>$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)', '..', '..', 'src'))</TemplatesSourceDirectory>
 		<UnoSdkDirectory>$([MSBuild]::NormalizePath('$(TemplatesSourceDirectory)', 'Uno.Sdk'))</UnoSdkDirectory>
+		<UnoSdkUpdaterDirectory>$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)'))</UnoSdkUpdaterDirectory>
```

Comment:
No need for NormalizePath if you're providing MSBuildThisFileDirectory. It should already be a normalized fully qualified path.